lib/mpl_toolkits/axes_grid1/axes_grid.py

```diff
@@ -433,28 +456,35 @@ def __init__(self, fig,
         """
         Parameters
         ----------
-        fig : Figure
+        fig : `.Figure`
             The parent figure.
         rect : (float, float, float, float) or int
             The axes position, as a ``(left, bottom, width, height)`` tuple or
             as a three-digit subplot position code (e.g., "121").
-        direction : {"row", "column"}, defaults to "row"
-        axes_pad : float or (float, float), defaults to 0.02
+        direction : {"row", "column"}, default: "row"
+        axes_pad : float or (float, float), default: 0.02
             Padding or (horizontal padding, vertical padding) between axes, in
             inches.
-        add_all : bool, defaults to True
-        share_all : bool, defaults to False
-        aspect : bool, defaults to True
-        label_mode : {"L", "1", "all"}, defaults to "L"
-        cbar_mode : {"each", "single", "edge", None }, defaults to None
-        cbar_location : {"left", "right", "bottom", "top"}, defaults to "right"
-        cbar_pad : float, defaults to None
-        cbar_size : size specification (see `.Size.from_any`), defaults to "5%"
-        cbar_set_cax : bool, defaults to True
+        add_all : bool, default: True
+        share_all : bool, default: False
+        aspect : bool, default: True
+       label_mode : {"L", "1", "all"}, default: "L"
```

Comment:
I suspect this is the offending line for the tests to fail. Appart it looks good.